.github/workflows/intel-oneapi.yml

```diff
@@ -0,0 +1,167 @@
+        export CC=mpiicc
+        export FC=mpiifort
+        export CPPFLAGS=-I${HOME}/netcdf/include
+        export LDFLAGS=-L${HOME}/netcdf/lib
+        wget https://github.com/Unidata/netcdf-fortran/archive/v4.5.3.tar.gz &> /dev/null
+        tar -xzf v4.5.3.tar.gz
+        pushd netcdf-fortran-4.5.3
+        ./configure --prefix=${HOME}/netcdf
+        make -j2
+        make install
+
+    - name: cache-esmf
+      id: cache-esmf
+      uses: actions/cache@v2
+      with:
+        path: ~/esmf
+        key: esmf-${{ runner.os }}-8.0.1-intel
+
+    - name: build-esmf
+      if: steps.cache-esmf.outputs.cache-hit != 'true'
+      run: |
+        pushd ~
+        export ESMF_DIR=~/esmf-ESMF_8_0_1
+        wget https://github.com/esmf-org/esmf/archive/ESMF_8_0_1.tar.gz &> /dev/null
+        tar zxf ESMF_8_0_1.tar.gz
+        cd esmf-ESMF_8_0_1
+        export ESMF_COMM=intelmpi
+        export ESMF_INSTALL_BINDIR=bin
+        export ESMF_INSTALL_LIBDIR=lib
+        export ESMF_INSTALL_MODDIR=mod
+        export ESMF_COMPILER=intel
+        export ESMF_INSTALL_PREFIX=~/esmf
+        export ESMF_NETCDF=split
+        export ESMF_NETCDF_INCLUDE=${HOME}/netcdf/include
+        export ESMF_NETCDF_LIBPATH=${HOME}/netcdf/lib
+        make -j2
+        make install
+
+    - name: cache-jasper
+      id: cache-jasper
+      uses: actions/cache@v2
+      with:
+        path: ~/jasper
+        key: jasper-${{ runner.os }}-2.0.25-intel
+
+    - name: build-jasper
+      if: steps.cache-jasper.outputs.cache-hit != 'true'
+      run: |
+        wget https://github.com/jasper-software/jasper/archive/version-2.0.25.tar.gz &> /dev/null
+        tar zxf version-2.0.25.tar.gz
+        cd jasper-version-2.0.25
+        mkdir build-jasper && cd build-jasper
+        cmake .. -DCMAKE_INSTALL_PREFIX=~/jasper
+        make -j2
+        make install
+
+    - name: checkout-nceplibs
+      uses: actions/checkout@v2
+      with:
+        repository: NOAA-EMC/NCEPLIBS
+        path: nceplibs
+
+    - name: cache-nceplibs
+      id: cache-nceplibs
+      uses: actions/cache@v2
+      with:
+        path: ~/nceplibs
+        key: nceplibs-${{ runner.os }}-1.3.0-intel
+
+    - name: build-nceplibs
+      if: steps.cache-nceplibs.outputs.cache-hit != 'true'
+      run: |
+        export ESMFMKFILE=~/esmf/lib/esmf.mk
+        wget https://github.com/NOAA-EMC/NCEPLIBS/archive/v1.3.0.tar.gz &> /dev/null
+        tar zxf v1.3.0.tar.gz
+        cd NCEPLIBS-1.3.0
+        mkdir build && cd build
+        cmake .. -DCMAKE_PREFIX_PATH='~;~/jasper;~/netcdf' -DCMAKE_INSTALL_PREFIX='~/nceplibs' -DFLAT=ON
+        make -j2
+       
+    - name: checkout-ufs-utils
+      uses: actions/checkout@v2
+      with: 
+        path: ufs_utils
+
+    - name: build
+      run: |
+        export ESMFMKFILE=~/esmf/lib/esmf.mk
+        cd ufs_utils
+        mkdir build && cd build
+        cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH='~;~/jasper;~/nceplibs;~/netcdf' -DENABLE_DOCS=On
```

Comment:
We only need to ENABLE_DOCS on one build, the GNU build I would suggest. It adds a lot of output which makes scrolling through it a bit of a pain...